Export Analysis Reports/Sdl.Community.ExportAnalysisReports/Helpers/Help.cs

```diff
@@ -1,221 +1,41 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Globalization;
+﻿using System.Collections.Generic;
+using System.ComponentModel;
 using System.IO;
-using System.Linq;
-using System.Xml;
-using Newtonsoft.Json;
 using Sdl.Community.ExportAnalysisReports.Model;
-using Sdl.Community.Toolkit.Core.Services;
-using Sdl.ProjectAutomation.Core;
-using Sdl.ProjectAutomation.FileBased;
 
 namespace Sdl.Community.ExportAnalysisReports.Helpers
 {
-	public static class Help
+	public class Help
 	{
-		public static readonly Log Log = Log.Instance;
-		public static readonly string CommunityFolderPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "SDL Community", "ExportAnalysisReports");
-		public static readonly string JsonPath = Path.Combine(CommunityFolderPath, "ExportAnalysisReportSettings.json");
-
-		public static string GetStudioProjectsPath()
-		{
-			try
-			{
-				var shortStudioVersion = GetInstalledStudioShortVersion();
-				if (string.IsNullOrEmpty(shortStudioVersion))
-				{
-					return string.Empty;
-				}
-
-				var projectsPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments), $@"Studio {shortStudioVersion}\Projects\projects.xml");
-				return projectsPath;
-			}
-			catch (Exception ex)
-			{
-				Log.Logger.Error($"GetStudioProjectsPath method: {ex.Message}\n {ex.StackTrace}");
-			}
-			return string.Empty;
-		}
-
-		public static Dictionary<string, LanguageDirection> LoadLanguageDirections(XmlDocument doc)
+		public void CreateDirectory(string path)
 		{
-			var languages = new Dictionary<string, LanguageDirection>();
-			try
-			{
-				var languageeDirectionNode = doc.SelectNodes("/Project/LanguageDirections/LanguageDirection");
-
-				if (languageeDirectionNode == null) return languages;
-				foreach (var item in languageeDirectionNode)
-				{
-					var node = (XmlNode)item;
-					if (node.Attributes == null) continue;
-					var lang = new LanguageDirection
-					{
-						Guid = node.Attributes["Guid"].Value,
-						TargetLang = CultureInfo.GetCultureInfo(node.Attributes["TargetLanguageCode"].Value)
-					};
-					languages.Add(lang.Guid, lang);
-				}
-			}
-			catch (Exception ex)
-			{
-				Log.Logger.Error($"LoadLanguageDirections method: {ex.Message}\n {ex.StackTrace}");
-			}
-			return languages;
+			Directory.CreateDirectory(path);
 		}
 
-		public static void LoadReports(XmlDocument doc, string projectPath, ProjectDetails project)
+		public Dictionary<string, bool> AddToDictionary(BindingList<LanguageDetails> languages)
```

Comment:
Indeed, the method was changed and it can be seen the last commit done on friday
![image](https://user-images.githubusercontent.com/13149323/81525125-5de84980-935c-11ea-91f8-7c284657235b.png)